docs/advanced-features/tabulator.md

```diff
@@ -106,11 +108,27 @@ order to query those tables.  This can be done by users via the per-bucket
 5. **Athena VPC**: If you are using a VPC endpoint for Athena, you must ensure
    it is accessible from the Quilt stack and Tabulator lambda.
 
+### Unrestricted Access
+
+> Available since Quilt Platform version 1.57
+
+By default, Tabulator is only accessible via a session provided by the Quilt Catalog,
```

Comment:
you also can use quilt3 for this
or Quilt Catalog == registry here?

Reply:
well, basically, yes